docs/source/markdown/podman-system-reset.1.md

```diff
@@ -7,7 +7,16 @@ podman\-system\-reset - Reset storage back to initial state
 **podman system reset** [*options*]
 
 ## DESCRIPTION
-**podman system reset** removes all pods, containers, images and volumes. Must be run after changing any of the following values in the `containers.conf` file: `static_dir`, `tmp_dir` or `volume_path`.
+**podman system reset** removes all pods, containers, images and volumes. Must
+be run after changing any of the following values in the `containers.conf` file:
+`static_dir`, `tmp_dir` or `volume_path`.
+
+Note: If administrators wants to change the /etc/containers/storage.conf in any
```

Comment:
Should we change this to "change any paths or the storage driver" instead of "change anything"? I'm pretty sure that setting ignore_chown_errors wouldn't need a `system reset` first, for example?

Reply:
I do you the term "may".  I think it is fairly difficult to get specific here.  I think they would only get here if something is majorly wrong.

Reply:
Ok I reworked a little bit more.